django/db/models/query_utils.py

```diff
@@ -91,6 +93,20 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
         query.promote_joins(joins)
         return clause
 
+    def deconstruct(self):
+        path = '%s.%s' % (self.__class__.__module__, self.__class__.__name__)
+        args, kwargs = (), {}
+        if len(self.children) == 1:
+            child = self.children[0]
+            kwargs = {child[0]: child[1]}
```

Comment:
Technically may be able to make this break, although it might not have any reason to ever actually happen in the wild.